src/simpleNbox.cpp

```diff
@@ -94,6 +96,7 @@ void SimpleNbox::init( Core* coreptr ) {
 
     // Register the inputs we can receive from outside
     core->registerInput(D_FFI_EMISSIONS, getComponentName());
+    core->registerInput(D_DACCS_UPTAKE, getComponentName());
```

Comment:
Should DACCS also be registered as a capability?

Reply:
I don't think so; FFI and LUC aren't registered that way. It's an input.

Reply:
Ah I thought that everything that passed into the getdata also had to be registered as a capability. Sound good.